marketplaceItems/appreciate/appResources/appData/appreciate_app.js

```diff
@@ -0,0 +1,947 @@
+/*
+    Appreciate
+    Created by Zach Fox on 2019-01-30
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+(function () {
+    // *************************************
+    // START UTILITY FUNCTIONS
+    // *************************************
+    // #region Utilities
+    var MS_PER_S = 1000;
+    var CM_PER_M = 100;
+    var HALF = 0.5;
+
+    // Returns the world position halfway between the user's hands
+    function halfwayBetweenHands() {
+        var leftHandPosition = MyAvatar.getJointPosition("LeftHandMiddle2");
+        if (!leftHandPosition) {
+            leftHandPosition = MyAvatar.getJointPosition("LeftHand");
+        }
+        var rightHandPosition = MyAvatar.getJointPosition("RightHandMiddle2");
+        if (!rightHandPosition) {
+            rightHandPosition = MyAvatar.getJointPosition("RightHand");
+        }
+
+        var centerPosition = Vec3.sum(leftHandPosition, rightHandPosition);
```

Comment:
Maybe this variable could be called something else since it's a little misleading until the following line(we could really use an instantiable Vec class for this exact thing).

Reply:
i'm not concerned about the name of this particular variable - i agree that it becomes clear when reading the following line